packages/inspector/src/Tree.tsx

```diff
@@ -101,16 +110,19 @@ export default function Tree({
     }
 
     return (
-      <StateKey value="p">
+      <>
         {propsToRender.map((prop, index) => {
           const val = data[prop];
           return (
-            <StateKey
-              key={typeof prop === "string" ? prop : index}
-              value={prop.toString()}
-            >
-              <Tree name={prop.toString()} data={val} parent={data} />
-            </StateKey>
+            <Tree
+              key={index}
+              name={prop.toString()}
+              data={val}
+              parent={data}
+              path={[...path, index]}
```

Comment:
This should be `[...path, prop]`, not `[...path, index]`

Reply:
Right, makes sense. Fixed.